### PR TITLE
Death of the Miner Worm

### PR DESCRIPTION
**Pull request by NPC1314**
## About The Pull Request

Couple fixes to bugs and a few issues that came up in testing

* Mining can't be done laying down no more (at least proper turf mining).
* Bow onmob sprites offsets tweaked a little, had a slight floating there going on
* Keffiyeh made into a proper coif style headpiece, means it can be worn on neck too and have a little armor, or be worn under.
Zyb merc gets it back now under the helmet instead of standard coif, looks pretty cool.
* Broken man beard had back sprite issues that made it look ass when facial hair flags werent toggled. Fixed.
* New blowgun sprite and upsized the dart a little. The noodle thing was too hard to click for one and looks like a stick.
Both the new blowgun and the upsized dart a lot easier to click.
blowie

Class tweaks of things thats been bugging me

* Totod had bunch of bugs, surcoat armor layering for one, sorted it with the proper items in proper slots  The patron enforcing was also busted, made a new one that works so you can actually play astrata or necra crusader now.
The gendered items had a oversight where silver sword wasnt assigned correctly, fixed
TOTOD NOT BUGGY

* Emir had some very strange things, pilgrim with top tier heavy armor (but no heavy armor training), loads of dosh and a good sword + swordskill 3 is a bit much, and lacked a bit of zyb flair. Solution - > replaced armor with arming jacket (no other role has it, noble themed gamb for both genders), added purple tunic and cloak so they look distinct but not just pure red since thats also heartfelt, all nobles being full red is kinda silly. New medallion name so you know its not just some zyb merc.
EMIR ARMOR NOW NOT BADNESS

* Amazon EDIT no point changing stats or traits since its slated for a rework anyways, but fixing the spear, adding shortsword and blowgun is plenty to make it feel a little more different for now.. 
AMAZON GEAR 

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Review by Optimism333 — COMMENTED** [review]
image

Endurace needs to be corrected to endurance here. Other than that, the changes look good.

File: code/modules/jobs/job_types/roguetown/adventurer/types/special/crusader.dm [hunk 1]
```diff
@@ -19,36 +20,56 @@
...
 	backr = /obj/item/rogueweapon/shield/tower/metal
-	backl = /obj/item/storage/backpack/rogue/satchel
+	backl = /obj/item/clothing/cloak/cape/crusader
 	belt = /obj/item/storage/belt/rogue/leather/plaquesilver
 	beltl = /obj/item/rogueweapon/sword/silver
+	backpack_contents = list(/obj/item/storage/belt/rogue/pouch/coins/rich = 1)
 
 	// Run this first so that patron gets changed before assigning gear.
-	if(H.patron != /datum/patron/divine/astrata || H.patron != /datum/patron/divine/necra)
+	if(H.patron == /datum/patron/divine/noc)
```
Comment (ThePainkiller) [inline code/modules/jobs/job_types/roguetown/adventurer/types/special/crusader.dm:38]: Wait, what's with this weird logic...?
If we picked Noc, we become forced Astrata faithful???????
If we picked Eora, or Faithless, we become forced Necra faithful???
If we picked Pestra, Dendor, Malum, Xylix, Ravox or Abyssor, we become forced Psydon faithful

W-why...??? Wat???

image

Comment (NPC1314) [reply code/modules/jobs/job_types/roguetown/adventurer/types/special/crusader.dm:38]: The lore and the guy who changed them from crusader to Totod tried to lock them to the three faiths of that order, but instead that old proc forced everyone to go Psydon.

after the change, choosing "not allowed" patrons gives a more random outcome, and astratan and necrans actually get to keep their faith.

Before :  Every patron got changed to Psydon
Now: The three intended ones get to keep their correct god, if you choose other gods its sorta random which of the three you get. Easy!

File: code/modules/jobs/job_types/roguetown/adventurer/types/special/crusader.dm [hunk 2]
```diff
@@ -68,19 +89,18 @@
 	H.change_stat("constitution", 2)
 	H.change_stat("strength", 1)
 	ADD_TRAIT(H, TRAIT_MEDIUMARMOR, TRAIT_GENERIC)
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/adventurer/types/special/crusader.dm:91]: Heavy armor trait covers medium armor, no?

Comment (BadAtThisGame302) [reply code/modules/jobs/job_types/roguetown/adventurer/types/special/crusader.dm:91]: Could replace it with Steelhearted maybe, idk.

File: code/modules/jobs/job_types/roguetown/adventurer/types/special/crusader.dm (hunk 1 above)
Comment (BadAtThisGame302) [reply code/modules/jobs/job_types/roguetown/adventurer/types/special/crusader.dm:38]: I was about to fix this but I am glad to see @ThePainkiller's faith selector issue was handled here.

File: code/modules/jobs/job_types/roguetown/adventurer/types/special/crusader.dm (hunk 2 above)
Comment (NPC1314) [reply code/modules/jobs/job_types/roguetown/adventurer/types/special/crusader.dm:91]: you are correct, heavy does help with medium too. 
Good idea with steel heart, supposed to be stoic knight after all

**Review by ThePainkiller — APPROVED** [review]
Yeah let's finally squash this silly exploit. I'm still unsure what was the logic behind the paladin's faith but I think I got it better understood now

**Pull request merged**